services/movie_session.py

```diff
@@ -0,0 +1,54 @@
+from django.db.models import QuerySet
+
+from db.models import CinemaHall, Movie, MovieSession
+
+
+def create_movie_session(
+        movie_show_time: str,
+        movie_id: int,
+        cinema_hall_id: int
+) -> MovieSession:
+
+    movie = Movie.objects.get(id=movie_id)
+    cinema_hall = CinemaHall.objects.get(id=cinema_hall_id)
```

Comment:
Using `CinemaHall.objects.get(id=cinema_hall_id)` will raise a `DoesNotExist` exception if no cinema hall is found with the given ID. Consider using `CinemaHall.objects.filter(id=cinema_hall_id).first()` to handle cases where the cinema hall might not exist.